include/aos/sm/resourcemanager.hpp

```diff
@@ -336,9 +336,9 @@ class ResourceManager : public ResourceManagerItf, private NonCopyable {
     StaticString<cNodeTypeLen> mNodeType;
     StaticString<cFilePathLen> mConfigPath;
     Error                      mConfigError {ErrorEnum::eNone};
-    NodeConfig                 mConfig;
+    UniquePtr<NodeConfig>      mConfig;
 
-    StaticAllocator<sizeof(StaticString<cNodeConfigJSONLen>)> mAllocator;
+    mutable StaticAllocator<sizeof(StaticString<cNodeConfigJSONLen>) + 2 * sizeof(NodeConfig)> mAllocator;
```

Comment:
Remove mutable and adjust size.

Reply:
Mutable is needed to allocate tmp node info object in 
`Error ResourceManager::CheckNodeConfig(const String& version, const String& config) const`